packages/blocks/src/multiple-choice/edit.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { InnerBlocks, RichText } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useClientId } from '@crowdsignal/hooks';
+
+/**
+ * Style dependencies
+ */
+import { MultipleChoiceWrapper } from './styles/multiple-choice';
+
+const ALLOWED_ANSWER_BLOCKS = [
+	'core/image',
+	'core/paragraph',
+	'crowdsignal-forms/poll-answer',
+];
+
+const MultipleChoice = ( props ) => {
+	const { attributes, setAttributes } = props;
+
+	useClientId( props );
+
+	const handleChangeQuestion = ( question ) => setAttributes( { question } );
+
+	const handleChangeNote = ( note ) => setAttributes( { note } );
+
+	return (
+		<MultipleChoiceWrapper>
+			<RichText
+				tagName="h3"
+				placeholder={ __( 'Enter your question', 'blocks' ) }
+				onChange={ handleChangeQuestion }
+				value={ attributes.question }
+			/>
+			<RichText
```

Comment:
Not sure if we should keep this 🤔 
Like, you'd just use a regular paragraph block if you deem it necessary.

Reply:
Good point, headlines do have a specific use, not sure why I did this.